helm-charts/stable/cloudwatch-agent/values.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+fullnameOverride: ''
+nameOverride: ''
+
+region: 'us-west-2'
+
+clusterName: 'my-eks-cluster'
+
+cloudwatchAgent:
+  image: 'amazon/cloudwatch-agent:1.247346.0b249609'
```

Comment:
Is this a valid image? If not, let's replace with a fake image like we do here: https://github.com/aws/eks-distro-build-tooling/blob/main/helm-charts/stable/amazon-eks-pod-identity-webhook/values.yaml#L22

Reply:
I think this is valid.
https://hub.docker.com/layers/amazon/cloudwatch-agent/1.247346.0b249609/images/sha256-ab6f27f16cfac6f2d307732fdf762f87baf57e2e4a530ec9e77a60855e8a99ea?context=explore